src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, GetArea {
```

Comment:
Don't begin class or method implementation with an empty line. The empty line after the class declaration on line 3 should be removed.